bundles/org.openhab.binding.flume/README.md

```diff
@@ -0,0 +1,120 @@
+# Flume Binding
+
+This binding will interface with the cloud API to retrieve water usage from your [Flume](https://flumewater.com/) water monitor.
+
+## Introduction
+
+The Cloud Connector is required as a "bridge" to interface to the cloud service from Flume.
+While the Flume API supports a rich querying of historical usage data, this binding only retrieves the cumulative water used and instantaneous water used, thus relying on Openhab's rich persistence services for exploring historical values.
+
+## Supported Things
+
+This binding supports the following things:
+
+| thing                     | type          | description                  |
+|----------                 |--------       |------------------------------|
+| Flume Cloud Connector     | Bridge        | This represents the cloud account to interface with the Flume API.  |
+| Flume Meter Device        | Device        | This interfaces to a specific Flume water monitor associated with the account. |
+
+This binding should work with multiple Flume monitors associated with the account, however it is currently only tested with a single device.
+
+## Discovery
+
+Once a Flume Cloud Connector is created and established, the binding will automatically discover any Flume Meter Devices' associated with the account.
+
+## Flume Cloud Connector (Bridge) Configuration
+
+The only configuration required is to create a Flume Cloud Connector thing and fill in the appropriate configuration parameters.
+The client id and client secret can be found under Settings/API access from the [Flume portal online](https://portal.flumewater.com/settings).
+Note, there is a rate limit of 120 queries per hour imposed by Flume so use caution when selecting the Refresh Interfacl.
+
+| Name                          | Type      | Description                                                                           | Default | Required | Advanced |
```

Comment:
Can you add a column for the `id` of the parameter e.g `userName`, `clientId` etc.